src/os2datascanner/projects/admin/tests/test_miniscanner.py

```diff
@@ -0,0 +1,135 @@
+from os2datascanner.projects.admin.adminapp.views.miniscanner_views import *
+from random import choice
+
+class DummyFile():
+    def __init__(self, name, fileContents) -> None:
+        self.contents = fileContents
+        self.name = name
+
+    def read(self):
+        return self.contents.encode()
+
+class FilesObj():
+    def __init__(self, objs) -> None:
+        self.files = objs
+    def get(self, obj):
+        return self.files[obj]
+    
+class Post():
+    def __init__(self, objs) -> None:
+        self.items = objs
+    def get(self, obj):
+        return self.items[obj]
+    
+class Meta():
+    def __init__(self, objs) -> None:
+        self.items = objs
+    def get(self, obj):
+        return self.items[obj]
+
+class Request():
+    def __init__(self, fileObjsDict, postObjsDict, metaObjsDict) -> None:
+        self.FILES = FilesObj(fileObjsDict)
+        self.POST = Post(postObjsDict)
+        self.META = Meta(metaObjsDict)
+
+def generateDummyContent(n=10):
+    CHARACTERS = "abcdefghijklmnopqrstuvwxyzæøå"
+    return '\n'.join([''.join([choice(CHARACTERS) for _ in range(25)]) for _ in range(n)]) # Generates a block of random characters
+
+cpr_rule = '{"name": null, "type": "cpr", "blacklist": ["p-nummer", "p.nr", "p-nr", "customer no", "tullstatistisk", "dhk:tx", "test report no", "tullstatistik", "faknr", "order number", "fakturanummer", "protocol no.", "ordrenummer", "customer-no", "fak-nr", "pnr", "bilagsnummer"], "whitelist": ["cpr"], "exceptions": "", "modulus_11": true, "sensitivity": null, "examine_context": true, "ignore_irrelevant": true}'
+customRegexRule = '{"type":"regex","expression":".*doesthiswordexist"}'
```

Comment:
You _do_ have the engine2 package available, so there's no need to have these JSON blobs lying around:

```python
import json
from os2datascanner.engine2.rules.cpr import CPRRule
from os2datascanner.engine2.rules.regex import RegexRule

...

cpr_rule = json.dumps(CPRRule().to_json_object())
customRegexRule = json.dumps(RegexRule("doesthiswordexist").to_json_object())
```

Reply:
Thanks for the feedback, i wasn't familiar with the engine2 package, but it looks much nicer like this. Fix upcoming !

Reply:
Fixed !